tests/property/ic_api/performance_counter/test/tests.ts

```diff
@@ -0,0 +1,231 @@
+import { defaultParams, expect, getCanisterActor, it, Test } from 'azle/test';
+import fc from 'fast-check';
+
+import { _SERVICE as Actor } from './dfx_generated/canister/canister.did';
+
+export function getTests(): Test {
+    return () => {
+        it('should calculate performanceCounter(0) instructions accurately from a query method', async () => {
+            const actor = await getCanisterActor<Actor>('canister');
+
+            await fc.assert(
+                fc.asyncProperty(
+                    fc.nat({
+                        max: 100 // Our algorithm for determinstically checking the number of instructions based on the number of loops breaks down soon after 100 iterations
```

Comment:
determinstically should be deterministically

Reply:
Fixed